pkgs/by-name/ha/haproxy/package.nix

```diff
@@ -18,14 +19,20 @@
 }:
 
 assert lib.assertOneOf "sslLibrary" sslLibrary [
+  "aws-lc"
```

Comment:
Perhaps it would be best if you could either add it to the bottom or the list or sort the entire array alphabetically
No preference which of those.

Reply:
I opted for the alphabetic sort. Thanks for reviewing so quickly!

Reply:
Excellent, thanks! My PRs get stuck quite often waiting for reviewers and HAProxy is quite dear to me, so two good reasons to get to it right away. :)